packages/medusa/src/api/routes/admin/orders/index.ts

```diff
@@ -294,6 +297,8 @@ export type AdminOrdersListRes = PaginatedResponse & {
   orders: Order[]
 }
 
+export const defaultAdminOrdersIncludesFields = ["returnable_items"]
```

Comment:
todo: I wouldn't make this a default field. The returnable items are only relevant in RMA flows right now, so no need to do add more load than needed on all other calls.

Reply:
Looks like this was missed in the rebase. Should not be a default field 👍